zypp-plugin/testsuite/tap-driver.sh

```diff
@@ -0,0 +1,651 @@
+#! /bin/sh
```

Comment:
Isn't this script provided by any other package (e.g., automake)?

Reply:
Yes and no, good point, I'll improve the commit message to explain.

Reply:
Done: https://github.com/openSUSE/snapper/pull/501/commits/c804a7af8d56f1a84e90745ca1c032d285237418

Reply:
Done:  https://github.com/openSUSE/snapper/pull/501/commits/c804a7af8d56f1a84e90745ca1c032d285237418